src/test/java/de/retest/recheck/cli/subcommands/DiffIT.java

```diff
@@ -205,8 +205,6 @@ public void diff_should_print_warning_when_exclude_option_with_invalid_filters()
 		GoldenMasterCreator.createGoldenMasterFile( gm1, true );
 		GoldenMasterCreator.createGoldenMasterFile( gm2, false );
 
-		final File outputFile = new File( outputDir, RecheckProperties.AGGREGATED_TEST_REPORT_FILE_NAME );
```

Comment:
This is a bit of a drive-by change. Maybe split this out into an additional clean-up commit?